routes/api/project/location/vm.rb

```diff
@@ -23,6 +23,15 @@ class CloverApi
         boot_image: r.params["boot_image"]
       )
 
+      UsageRecord.create_with_id(
```

Comment:
I wonder if this should be created once the resource is up and running. In case the provisioning fails or stuck, we would create a billing record for nothing. Maybe this is good for now but food for thought.